pages/Configuring/Multi-GPU.md

```diff
@@ -0,0 +1,132 @@
+# Table of contents
+
+{{< toc >}}
+
+# General
+
+If your host machine uses multiple GPUs, you may want to primarily use a GPU for rendering all the elements for Hyprland including windows, animations, hardware acceleration for certain applications, etc.
+
+This setup is very common in the likes of gaming laptops, GPU-passthrough(without VFIO) capable hosts, and if you have multiple GPUs in general.
+
+# Detecting GPUs
+
+For this case, the writer is taking the example of their laptop. Here are some of the details for the same
+
+Upon running `lspci | grep VGA`, I can list all the video devices installed on this laptop
+
+```
+01:00.0 VGA compatible controller: NVIDIA Corporation TU117M [GeForce GTX 1650 Mobile / Max-Q] (rev a1)
+06:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Cezanne [Radeon Vega Series / Radeon Vega Mobile Series] (rev c6)
+```
+
+Here it is clear that the writer has 2 GPUs, the dedicated NVIDIA GTX 1650 Mobile / Max-Q and the integrated AMD Cezanne Radeon Vega Series GPU.
+
+So this would indicate, Linux should be detecting 2 GPUs, and thus 2 "cards" in `/dev/dri/`. Let us verify this using `ls /dev/dri/`
+
+```
+by-path  card0  card1  renderD128  renderD129
+```
+
+Here, `card0` and `card1` correspond to `renderD128` and `renderD129` respectively. So for every card you have installed on your host device the format would be something like this:
+
+` cardN  renderD(128+N)`
+
+We now need to determine which of these `card0` and card1` correspond to the AMD and NVIDIA GPU.
```

Comment:
You don't need to jump through hoops to check which device is which card. Simply `ls -l /dev/dri/by-path`, and the PCI ID will show up as a link to its respective card.

This also means you can get rid of the same-vendor section as well.